app/templates/frameworks/services.html

```diff
@@ -65,91 +64,115 @@ <h2 class="summary-item-heading">{{ framework.name }} is closed for applications
-  {% elif framework.status == 'pending' %}
-    <p class="hint">These services were not completed</p>
-  {% endif %}
-  {% call(draft) summary.list_table(
-    drafts,
-    caption="Draft services",
-    empty_message="You haven’t added any services yet." if framework.status == 'open' else "You didn’t add any services.",
-    field_headings=[
-        "Service name",
-        "Progress",
-        "Make a copy"
-    ],
-    field_headings_visible=False
-  ) %}
-    {% call summary.row() %}
-      {{ summary.service_link(draft.serviceName,
-                              url_for(".view_service_submission", framework_slug=framework.slug, lot_slug=draft.lot, service_id=draft.id)) }}
+  {% set ns = namespace(draft_rows = []) %}
+  {% for draft in drafts %}
+    {% set service_link %}
+    <a class="govuk-link" href="{{ url_for('.view_service_submission', framework_slug=framework.slug, lot_slug=draft.lot, service_id=draft.id) }}">{{ draft.serviceName }}</a>
+    {% endset %}
+    {% set make_copy_button %}
+    <form method="post" action="{{ url_for('.copy_draft_service', framework_slug=framework.slug, lot_slug=draft.lot, service_id=draft.id) }}">
+      <input type="hidden" name="csrf_token" value="{{ csrf_token_value or csrf_token() }}" />
+    {{ govukButton({
+      "text": "Make a copy",
+      "classes": "govuk-!-margin-0",
+    }) }}
+    </form>
+    {% endset %}
 
-      {% if framework.status == 'open' %}
-        {{ summary.text(submission.multiline_string(
-          submission.can_be_completed_text(draft.unanswered_required, framework.status),
+    {% set row = ns.draft_rows.append(
+      [
+          {"html": service_link},
+          {"html": submission.multiline_string(submission.can_be_completed_text(draft.unanswered_required, framework.status),
           submission.unanswered_required_text(draft.unanswered_required, draft.unanswered_optional),
-          submission.unanswered_optional_text(draft.unanswered_required, draft.unanswered_optional)
-        )) }}
-        {% call summary.field(action=True) %}
-          <form method="post" action="{{ url_for('.copy_draft_service', framework_slug=framework.slug, lot_slug=draft.lot, service_id=draft.id) }}">
-            <input type="hidden" name="csrf_token" value="{{ csrf_token_value or csrf_token() }}" />
-            {{ govukButton({
-              "text": "Make a copy",
-              "classes": "govuk-!-margin-0",
-            }) }}
-          </form>
-        {% endcall %}
-      {% else %}
-        {{ summary.text() }}
-        {{ summary.text() }}
-      {% endif %}
-    {% endcall %}
-  {% endcall %}
+          submission.unanswered_optional_text(draft.unanswered_required, draft.unanswered_optional))},
+          {"html": make_copy_button},
+      ]
+    ) %}
+{% endfor %}
+<p class="summary-item-top-level-action">
+  <a href="{{ url_for(".start_new_draft_service", framework_slug=framework.slug, lot_slug=lot.slug) }}" class="summary-change-link">Add a service</a>
+</p>
```

Comment:
I just copied this markup to replicate what the previous macro was doing, not sure if we want to keep this or replace with something else like a button?

Reply:
I think it makes sense to have a button to add services, since when you're on this page in the right framework state, that's your main action.

Depends on whether we think we should tie that action to the Drafts Services table - I don't think we need to.

Reply:
Could we move the CTA directly below the `<h1>`, or do you think it's better placed where it is?

Reply:
Have implemented this suggestion in 78d57f0